examples/game_of_life/README.md

```diff
@@ -0,0 +1,66 @@
+# OpenCensus Game of Life Codelab
+
+This codelab demonstrates OpenCensus with a simple client/server Java example based on
+[Conway's Game of Life](https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life)
+(abbreviated as Life for the remainder of this Codelab). You will learn how to define OpenCensus
+tags, measures and views, how to add tags to the Life application, how to record stats against
+current tag context, and how to view and understand the stats collected against those tags.
```

Comment:
We should include traces as well.

Reply:
I talked with @dinooliva earlier, and we were not sure if this is a good example for tracing. At this moment it's only demonstrating tags and stats APIs.

Reply:
I think GOL itself is sufficient to demonstrate some basic concept of tracing. We can create a client span to record the latency of a single rpc invocation, and add server span to further break it down and see how much time is spent on network or server calculation. Optionally, annotations could be added on each iteration.

We might need to explain the difference of using stats and trace to record the latency, e.g. the stats emphasize on aggregation while tracing emphasize on details of individual call, which will help user understand these concepts better.

Reply:
SGTM, I added some simple usage on tracing/spans. A detailed user tutorial is WIP.